politeiad/backendv2/tstorebe/plugins/pi/pi.go

```diff
@@ -23,6 +26,35 @@ var (
 	_ plugins.PluginClient = (*piPlugin)(nil)
 )
 
+// statusData is stored in an in-memory cache, it includes metadata which
+// won't change in the future to avoid expensive real-time evaluations.
+//
+// proposalStatus is optional and will be populated only if the proposal
+// status is not expected to change.
+//
+// voteStatus is optional and will be populated only for proposals with
+// vote status 'approved' to improve performance of retrieving proposal
+// statuses of approved proposals.
+type cacheData struct {
+	proposalStatus *pi.PropStatusT
+	voteStatus     *ticketvote.VoteStatusT
+}
+
+// piCacheLimit limits the number of entries in the piCache context.
+const piCacheLimit = 1000
```

Comment:
How was this number chosen?  What is the maximum size of a cache entry?  What is the total size of the cache based on this?